### PR TITLE
Add readme for Whisper Test

### PR DESCRIPTION
**Pull request by chtanch**
## Description

- Add readme for Whisper evaluation test.
- Modify script to support sourcing from online dataset and local dataset.
- Remove unused files and code

Sample results with whisper-base model (sym_int4):

image

### CODE REVIEW

File: python/llm/test/benchmark/whisper/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Whisper Test
+The Whisper Test allows users to evaluate the performance and accuracy of Whisper speech-to-text models.
+For accuracy, the model is tested on the [LibriSpeech] (https://huggingface.co/datasets/librispeech_asr) dataset using [Word Error Rate (WER)](https://github.com/huggingface/evaluate/tree/main/metrics/wer) metric.
```
Comment (lalalapotter) [inline python/llm/test/benchmark/whisper/README.md:3]: "[LibriSpeech] (https://huggingface.co/datasets/librispeech_asr)" there is a redundant space.

File: python/llm/test/benchmark/whisper/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Whisper Test
+The Whisper Test allows users to evaluate the performance and accuracy of Whisper speech-to-text models.
```
Comment (lalalapotter) [inline python/llm/test/benchmark/whisper/README.md:2]: Maybe add a whisper link here.

File: python/llm/test/benchmark/whisper/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Install Dependencies
+```bash
+pip install datasets evaluate soundfile librosa jiwer
+```
+
+## Run
+The LibriSpeech dataset contains 'clean' and 'other' splits. By default, the evaluation is done on the 'other' split.
+
+### Evaluation on CPU
```
Comment (lalalapotter) [inline python/llm/test/benchmark/whisper/README.md:14]: Maybe we can unify these two modules into one command, or put them in a shell script file. In fact, some environment variables may need to be set when running on the GPU, but we can make no distinction here. Taking CPU as an example, we will explain the meaning of the `device` field and let the user choose by themselves.

File: python/llm/test/benchmark/whisper/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+> If you get the error message `ConnectionError: Couldn't reach http://www.openslr.org/resources/12/test-other.tar.gz (error 403)`, you can source from a local dataset instead.
+
+## Using a local dataset
+By default, the LibriSpeech dataset is downloaded at runtime with Huggingface Hub. If you prefer to source from a local dataset instead, please set the following environment variable before running the evaluation script
+
+```bash
+export LIBRISPEECH_DATASET_PATH=/path/to/dataset_folder
+```
+
+Make sure the local dataset folder contains 'dev-other.tar.gz','test-other.tar.gz', and 'train-other-500.tar.gz'. The files can be downloaded from http://www.openslr.org/resources/12/
```
Comment (lalalapotter) [inline python/llm/test/benchmark/whisper/README.md:35]: Add some description for output details? WER, RTF, and the output of max WER prediction etc...

**Review by lalalapotter — COMMENTED** [review]
Others LGTM.

**Comment by hkvision** [comment]
Move the files to dev/benchmark folder

**Comment by chtanch** [comment]
Output with Whisper-base on CPU
image

**Review by lalalapotter — APPROVED** [review]
LGTM.

**Pull request merged**